src/content/docs/infrastructure/amazon-integrations/connect/eks-add-on.mdx

```diff
@@ -19,7 +19,7 @@ Amazon Elastic Kubernetes Service (Amazon EKS) is a managed Kubernetes service t
 
 ## Prerequisites [#prereq]
 
-* An [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) cluster. See the user guide [for more information](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+* An [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) cluster. For more information, see our [Kubernetes integration: compatibility and requirements](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements/) and [Amazon's user guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
```

Comment:
I'm not sure you need the "Kubernetes integration:" in the link, maybe cutting it and adding "doc" to the end of "compatibility and requirements" would flow better? Just a nit pick really, feel free to ignore.

Reply:
Thanks, Jeff!